src/gui/qgsrasterlayersaveasdialog.cpp

```diff
@@ -169,7 +169,7 @@ QgsRasterLayerSaveAsDialog::QgsRasterLayerSaveAsDialog( QgsRasterLayer *rasterLa
   else
   {
     mFilename->setStorageMode( QgsFileWidget::SaveFile );
-    mFilename->setDialogTitle( tr( "Select output file" ) );
+    mFilename->setDialogTitle( tr( "Save Layer as ..." ) );
```

Comment:
Remove the space before ... ? (In order to have same string and a unique text  to translate)